### ISSUE TITLE
apparent problem with argument parsing - "-x" (success) vs "--xvfb" (failure)

### ISSUE THREAD

**Issue by gavinc** [adamn/python-webkit2png#19]
tl;dr -- "-x" works, "--xvfb" doesn't.

If I instantiate using:
 ./webkit2png.py --debug --log=arse.log --xvfb=1024 768 http://example.com

the resulting xvfb run command is repeating instantations of xvfb-run strings:
DEBUG:webkit2png:Executing xvfb-run --auto-servernum --server-num 1014620 --server-args=-screen 0, 1024x768x24 ./webkit2png.py --debug --log=arse.log --xvfb=1024 768 http://example.com

DEBUG:webkit2png:Executing xvfb-run --auto-servernum --server-num 1014690 --server-args=-screen 0, 1024x768x24 ./webkit2png.py --debug --log=arse.log --xvfb=1024 768 http://example.com

This repeating call never ends, and the PNG is never produced.

Whereas if I run the command:
./webkit2png.py --debug --log=arse.log -x 1024 768 http://example.com

then the command that is run is:
DEBUG:webkit2png:Executing xvfb-run --auto-servernum --server-num 1013165 --server-args=-screen 0, 1024x768x24 ./webkit2png.py --debug --log=arse.log http://example.com
DEBUG:webkit2png:Version 20091224, Python 2.6.5 (r265:79063, Apr 16 2010, 13:09:56)
[GCC 4.4.3], Qt 4.6.2
DEBUG:webkit2png:loading started
DEBUG:webkit2png:loading finished with result True
DEBUG:webkit2png:Processing result
DEBUG:webkit2png:contentsSize: PyQt4.QtCore.QSize(1020, 700)

And the PNG is produced - and all is well.

Looking at the code (I'm not a pythonist myself) - i suspect some issue around line 443 of the current version:
elif sys.argv[i] in ["-x", "--xvfb"]:

My rationale is that the bare "-x" is followed by a space and the integer:
-x 640 480

whereas the "--xvfb" is followed by an equals sign then immediately the integer:
--xvfb=640 480

So maybe it's not parsing it out as desired.

G
